docs/index.md

```diff
@@ -45,7 +45,9 @@ about pipeline inputs.
     "fasta": "A path to a reference fasta",
     "fai": "The path to the index associated with the reference fasta",
     "dict": "The path to the dict file associated with the reference fasta"
-  }
+  },
+  "pipeline.sample.Sample.library.Library.readgroup.Readgroup.qc.QC.Cutadapt.adapter": "The adapters to be cut from the forward reads",
+  "pipeline.sample.Sample.library.Library.readgroup.Readgroup.qc.QC.Cutadapt.adapterRead2": "The adapters to be cut from the reverse reads (if paired-end reads are used)"
```

Comment:
It might be good to explicitly state somewhere that no adapter clipping will be performed if these are not defined. I think most people (at least internally) would assume there is some default adapter(s) defined, as that was always the case historically.

Reply:
Should we maybe define a sane default at the pipeline level?

Reply:
Made a few changes to how we handle the QC pipeline and reworded the documentation example to include defaults.